pinakes/main/approval/models.py

```diff
@@ -184,10 +189,96 @@ class Meta:
             ),
         ]
 
+    def move_internal_sequence(self, delta):
+        """
+        Move current record up or down relative to other records.
+        delta is an integer
+        """
+        if delta == 0:
+            return
+
+        if delta > 0:
+            self.internal_sequence = Decimal(
+                self._internal_sequence_plus(delta)
+            )
+        else:
+            self.internal_sequence = Decimal(
+                self._internal_sequence_minus(-delta)
+            )
+        self.save()
```

Comment:
It's probably better to avoid implicit save of a model in a method that modifies it.